### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by curquiza**
related to https://github.com/meilisearch/devrel/issues/492

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
 
 ## What does this PR do?
 Fixes #<issue_number>
-<!-- Please link the issue you're trying to fix with this PR, if none then please create an issue first. -->
 
 ## PR checklist
 Please check if your PR fulfills the following requirements:
-- [ ] Does this PR fix an existing issue?
+- [ ] Does this PR fix an existing issue or do you have well-explained what you changed in the PR description?
```
Comment (brunoocasali) [inline pull_request_template.md:8]: Accordingly to Grammarly :D 

```suggestion
- [ ] Does this PR fix an existing issue, or do you have a well-explained what you changed in the PR description?
```

Comment (CaroFG) [reply pull_request_template.md:8]: I think the right thing would be to say:  Does this PR fix an existing issue, or do you have **well explained** what you changed in the PR description? (without the dash)

Comment (ferdi05) [reply pull_request_template.md:8]: `well` is super subjective, a little bit afraid by this

Comment (curquiza) [reply pull_request_template.md:8]: What would you read instead @ferdi05?
What I want is details, so maybe simple "have you detailed"

Comment (ferdi05) [reply pull_request_template.md:8]: If not, then we could say that any PR submitted without an existing issue would fall in the category of "PRs that don't need an issue", and think of another way to say this to contributors.

Comment (curquiza) [reply pull_request_template.md:8]: > I'm also wondering, do you think that some contributors would submit a PR which has a significant impact without creating an issue first?

It sometimes happened, despite the rules on the CONTRIBUTING.md file
In this case we discuss the "issue" on the PR, which is not really convenient, and depending on the repo, it often lead to open an issue

Comment (CaroFG) [reply pull_request_template.md:8]: What about: Have you listed the changes applied in the PR description (and why they are needed)?

Comment (curquiza) [reply pull_request_template.md:8]: Like this one!

Comment (curquiza) [reply pull_request_template.md:8]: WDYT?

**Comment by curquiza** [comment]
@ferdi05 @brunoocasali and @CaroFG 
I also created a dedicated section in the PR to encourage PR filling it!

**Review by ferdi05 — APPROVED** [review]
lgtm, but I don't want to be the only reviewer

**Comment by curquiza** [comment]
I also put @dichotommy as review since your are concerned

**Review by CaroFG — APPROVED** [review]

**Review by brunoocasali — APPROVED** [review]
Awesome! :)

**Review by dichotommy — APPROVED** [review]
LGTM 🌊🏄🏻

**Pull request merged**